src/q1/math-FSAB1101/summary/math-FSAB1101-summary.tex

```diff
@@ -1349,4 +1349,4 @@ \section{Série harmonique et nombres premiers}
   Ce qui est absurde car c'est une série harmonique.
 \end{proof}
 
-\end{document}
+\end{document}
```

Comment:
Tu saurais remplacer ce fichier (synthèse math 1) dans ton repo local celui qui est sur le github officiel, et puis commit ? Ca va permettre de supprimer ce changement inutile que tu as fait dans le fichier.

Reply:
Not a fan of this solution personally. Git has tools which can make this job much cleaner than copy-pasting an up-to-date file.

Reply:
Well, even for beginning people only knowing git add, commit and push? =)
Otherwise I fully agree, but hopefully "squash and merge" allows the admins to deal with these extra commits.